### PR TITLE
:lipstick:Video input redesign

### PR DESCRIPTION
**Pull request by Dancunrandy**
# Description

Re-designed input video  

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Screenshot (optional)
Screenshot from 2023-11-05 12-36-04
Screenshot from 2023-11-05 12-39-23

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**: (optional)
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: libs/features/convs-mgr/stories/blocks/library/main/src/lib/providers/video-input-block.jsplumb.ts [hunk 1]
```diff
@@ -21,9 +21,16 @@ export function _VideoInputBlockDecoratePlumb(block: VideoInputBlock, comp: Comp
...
+    maxConnections: -1
+  });
+  jsPlumb.addEndpoint(comp.location.nativeElement, {
+    // Whether the anchor is target (Other Block -> This Block)
+    target: true,
+    cssClass:"block_endpoint",
+    endpoint: "Dot",
+    anchor: [1,0.9,0,1],
     maxConnections: -1
   });
```
Comment (LemmyMwaura) [inline libs/features/convs-mgr/stories/blocks/library/main/src/lib/providers/video-input-block.jsplumb.ts:34]: remove this second jsplumb

File: libs/features/convs-mgr/stories/blocks/library/video-input-block/src/lib/components/video-input-block/video-input-block.component.html [hunk 2]
```diff
@@ -2,7 +2,7 @@
 	<form [formGroup]="videoInputForm" fxLayout="column" fxLayoutALign="start center">
 		<input [id]="videoInputId" formControlName="message" type="text" [placeholder]="'PAGE-CONTENT.BLOCK.PLACEHOLDER.VIDEO-INPUT' | transloco" />
 		
-		<app-default-option-field [jsPlumb]="jsPlumb" [blockFormGroup]="videoInputForm"></app-default-option-field>
+		<!-- <app-default-option-field [jsPlumb]="jsPlumb" [blockFormGroup]="videoInputForm"></app-default-option-field> -->
```
Comment (LemmyMwaura) [inline libs/features/convs-mgr/stories/blocks/library/video-input-block/src/lib/components/video-input-block/video-input-block.component.html:5]: Leave the default option for now.

**Review by LemmyMwaura — CHANGES_REQUESTED** [review]
Update your PR desc, Sync your changes then resolve conflicts

**Comment by Dancunrandy** [comment]
@LemmyMwaura , resolved conflicts and added the default option

**Comment by LemmyMwaura** [comment]
Closing this as its been resolved

**Pull request closed without merging**